### DISCUSSION TITLE
Estrarre attributo da climate

### DISCUSSION THREAD

**Discussion by echopage** [indomus/forum#1585] [HUB: Home Assistant]
Salve volevo estrarre l'attributo temperature da climate.netatmo_sala, ho provato con template, ma non supporta climate.  Come posso fare? Grazie

**Comment by snakuzzo** [comment]
Scusa ma questa cosa dove l'hai letta ? :D

```
{{ state_attr('climate.cameretta','temperature') }}
```

![111](https://user-images.githubusercontent.com/7205371/159126192-757c6953-0952-441b-927d-37c89a379660.png)

**Reply by echopage** [reply]
ho provato ma mi da errore template, provo a ricontrollare l'identazione!

**Reply by Scialla** [reply]
che poi l'integrazione netatmo espone direttamente le temperature:
(non mi fa aggiungere lo screenshoot, da errore)
